backend/tracim_backend/tests/functional/test_account.py

```diff
@@ -984,8 +985,8 @@ def test_api__get_account_workspaces__ok_200__nominal_case(
         app_api = application_api_factory.get()
```

Comment:
`get()` means "the object/instance already exist", otherwise it must be `create()` or `create_instance()`

Reply:
this is test related, all factory object for test use, ".get()" this refactoring should be done something else (not app related). see new issue #2627